src/material-experimental/mdc-button/_mdc-button.scss

```diff
@@ -39,6 +53,10 @@
         @include mdc-button-ink-color(on-error, $query: $mat-theme-styles-query);
         @include mdc-states-base-color(on-error, $query: $mat-theme-styles-query);
       }
+
+      &[disabled][disabled] {
```

Comment:
Can we avoid this? We have something like this in the current button and we've had complaints in the past. Otherwise it should at least have a comment about why we need the extra specificity.

Reply:
Can you add a TODO?